web/cobrands/buckinghamshire/assets.js

```diff
@@ -321,6 +321,7 @@ fixmystreet.assets.add(defaults, {
         }
     },
     no_asset_msg_id: '#js-not-a-road',
+    no_asset_msg_ids: '#js-not-a-road,#js-not-council-road-he,#js-not-council-road-other,#js-not-council-road',
```

Comment:
I'm a little confused why we have both of these. Is it because `no_asset_msg_id` is sometimes shown, but `no_asset_msg_ids` are only ever hidden? A comment explaining the difference, or a slightly more distinct name would be good for clarity when looking back later on.

Reply:
Yes, so if there is no `no_asset_msg_ids` it will just hide `no_asset_msg_id` by default. It's the group of IDs that could be shown (ie. for this Bucks asset road_not_found will show js-not-a-road but road_found will show one of the other IDs) so they need to be hidden at the same time because we don't know which one had been shown. I guess could rename to `no_asset_possible_msg_ids`, but comment here and in fixmystreet/assets.js probably clearer.

Reply:
I've switched it to a class which I think is hopefully clearer.